src/WebAPI/Models/RegisterModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace WebAPI.Models
+{
+    public class RegisterModel
+    {
+        [Required]
+        public string Login { get; set; }
+        [Required]
+        public string FirstName { get; set; }
+        [Required]
+        public string LastName { get; set; }
+        [Required]
+        [EmailAddress]
+        public string Email { get; set; }
+        [Required]
+        [MinLength(10)]
```

Comment:
Your users will be happy to know that they have to remember ten-symbol-length password :-D